alpine/3.15/8.0/Dockerfile

```diff
@@ -1,16 +1,13 @@
 # syntax = docker/dockerfile:1.0.2-experimental
 ARG SPRYKER_PHP_VERSION=8.0.26
```

Comment:
Could you please recheck the PHP version and the composer? 
Thanks.

Reply:
Updated